es-app/src/SystemData.h

```diff
@@ -8,23 +8,31 @@
 #include "PlatformId.h"
 #include "ThemeData.h"
 #include "FileFilterIndex.h"
+#include "CollectionSystemManager.h"
+
+struct SystemEnvironmentData
```

Comment:
OK, no problem. There are some other additional warnings, but none of them very interesting. Only this one seemed like one that could lead to trouble down the road.